script/build-win32.sh

```diff
@@ -57,12 +57,21 @@ git config --file $SYSTEM_CONFIG core.symlinks "false"
 git config --file $SYSTEM_CONFIG core.autocrlf "true"
 git config --file $SYSTEM_CONFIG core.fscache "true"
 git config --file $SYSTEM_CONFIG http.sslBackend "schannel"
+
 # See https://github.com/desktop/desktop/issues/4817#issuecomment-393241303
 # Even though it's not set openssl will auto-discover the one we ship because
 # it sits in the right location already. So users manually switching
 # http.sslBackend to openssl will still pick it up.
 git config --file $SYSTEM_CONFIG --unset http.sslCAInfo
 
+# Git for Windows 2.18.0 now supports controlling how curl uses any certificate
+# bundle - rather than just loading the bundle if http.useSSLCAInfo is set
+# For the moment we want to favour using the OS certificate store unless the
+# user has overriden this in their global configuration.
+#
+# details: https://github.com/dscho/git/commit/ffd4963bde17c11dfc2580d258a315964fa21378
+git config --file $SYSTEM_CONFIG http.schannel.useSSLCAInfo "false"
```

Comment:
As a heads up, this new config value won't actually address the reported issue, because Git uses `http.{url}.*` to denote URL-specific config overrides, and `schannel` here gets interpreted as a URL rather than a dedicated config entry.

https://github.com/git-for-windows/git/pull/1747 is the resolution of this issue, and hopefully will go out in an update :soon: